documentation/modules/proc-iot-api.adoc

```diff
@@ -27,23 +27,12 @@ endif::[]
 
 .Procedure
 
-. In the Hono project, run:
-+
-[options="nowrap",subs="attributes"]
-----
-cd cli
-----
-. Run the following command at least once:
```

Comment:
What does "at least once" mean? Assuming this step is to install the command-line client, suggest changing to: "Run the following command to install the command-line client:"

Reply:
This is now removed from the doc. It meant that you need to compile Hono command line source code at least once. With this PR we're downloading binary client, so no need for this anymore.